app/Console/Command/MoveEvents.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Console\Command;
+
+use Broadway\CommandHandling\CommandBus;
+use CultuurNet\UDB3\Event\Commands\UpdateLocation;
+use CultuurNet\UDB3\Event\ValueObjects\LocationId;
+use CultuurNet\UDB3\Search\ResultsGenerator;
+use CultuurNet\UDB3\Search\ResultsGeneratorInterface;
+use CultuurNet\UDB3\Search\SearchServiceInterface;
+use CultuurNet\UDB3\Search\Sorting;
+use Exception;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+
+/**
+ * @url https://jira.publiq.be/browse/III-6228
+ */
+class MoveEvents extends AbstractCommand
+{
+    private const PLACE_UUID = 'place-uuid';
+    private const QUERY = 'query';
+    private const FORCE = 'force';
+    private const DRY_RUN = 'dry-run';
+
+    private ResultsGeneratorInterface $searchResultsGenerator;
+
+    public function __construct(
+        CommandBus $commandBus,
+        SearchServiceInterface $searchService
+    ) {
+        parent::__construct($commandBus);
+        $this->searchResultsGenerator = new ResultsGenerator(
+            $searchService,
+            new Sorting('created', 'asc'),
+            100
+        );
+    }
+
+    public function configure(): void
+    {
+        $this
+            ->setName('event:move')
+            ->setDescription('Move an event from one place to another based on a SAPI3 query')
```

Comment:
I would write something like "Update the location of all events from the given SAPI3 query to the given new location"